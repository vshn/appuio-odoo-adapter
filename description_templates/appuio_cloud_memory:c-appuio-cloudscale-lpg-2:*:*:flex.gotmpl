{{- $class := "flex" -}}

{{template "_compute" dict "ComputeClass" $class "Item" . -}}
