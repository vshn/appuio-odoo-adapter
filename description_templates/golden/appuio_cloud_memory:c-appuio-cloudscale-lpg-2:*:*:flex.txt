Compute [flex]

Qty: 87955674.09 UNIT-Minutes
Unit Price: CHF 0.0000007460 / UNIT / Minute
--
Average Usage: 7.61 UNIT / Minute
↳ Memory Requests: 0.70 UNIT / Minute
↳ Compensation for excess CPU Requests: 2.23 UNIT / Minute
